### PR TITLE
m2_setup

### PR DESCRIPTION
**Pull request by recycletechno**
Init pull request

### CODE REVIEW

**Comment by Illania** [comment]
Looks good, approved!

**Review by Losyash — APPROVED** [review]
Good code.

File: patch_cpu.ps1 [hunk 1]
```diff
@@ -20,6 +20,11 @@ $search_replace = @{
     ".to('cuda')" = ".to(torch.device(`'cuda:0`' if torch.cuda.is_available() else `'cpu`'))";
 }
 
+if (!(Test-Path ".\SimSwap")) { Write-Host "Directory not found"; exit 1 }
```
Comment (Illania) [inline patch_cpu.ps1:23]: Please explain why do we need this line?

**Review by Illania — COMMENTED** [review]
please answer my questions

File: patch_cpu.ps1 (hunk 1 above)
Comment (Illania) [reply patch_cpu.ps1:23]: Screenshot 2023-04-17 at 12 21 21
 
When I run your modified script from deep_fake repo root folder, I get "directory not found" message.

File: demo/swapsingle/solod.jpeg
Comment (Illania) [inline demo/swapsingle/solod.jpeg:1]: Can you rename it to dst.jpeg please to follow same naming pattern as in other demo?

File: demo/swapsingle/soz_1sec.mp4
Comment (Illania) [inline demo/swapsingle/soz_1sec.mp4:1]: For same reason as I mentioned above it is better to rename this to "input.mp4" and please put an output.mp4 into same folder.

File: patch_cpu.ps1 (hunk 1 above)
Comment (Illania) [reply patch_cpu.ps1:23]: I think it is better to remove those changes from the script and just write in installation guide that you need to do the following: 
cd SimSwap
pwsh ../patch_cpu.ps1
It is in general a bad practice to make script manage the directory from where it should be run from.

**Comment by alduinsh** [comment]
I don't have additional questions for the code, please check the questions from Illania.

**Comment by alduinsh** [comment]
Sorry, open again for approval

**Review by alduinsh — COMMENTED** [review]
I don't have additional questions for the code, please check the questions from Illania.

File: patch_cpu.ps1 (hunk 1 above)
Comment (recycletechno) [reply patch_cpu.ps1:23]: What I need to do? Remove the code?

File: demo/swapsingle/solod.jpeg
Comment (recycletechno) [reply demo/swapsingle/solod.jpeg:1]: Renamed to dst.jpeg

File: demo/swapsingle/soz_1sec.mp4
Comment (recycletechno) [reply demo/swapsingle/soz_1sec.mp4:1]: Renamed to input.mp4

**Review by recycletechno — COMMENTED** [review]
What is "submit"?

File: patch_cpu.ps1 (hunk 1 above)
Comment (Illania) [reply patch_cpu.ps1]: yes, please

**Pull request merged**